.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Scala CI
+
+on:
+  push:
+    branches: [main]
+    tags: ["v*"]
+  pull_request:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+
+      - name: Checking formating
+        run: sbt scalafmtCheckAll
+
+      - name: Checking import order
+        run: sbt "scalafixAll --check"
```

Comment:
Actually... AFAIK this runs most (of not all) of `sbt compile` anyway... so I think the benefit of extracting `scalafmtCheckAll` and `scalafixAll --check` into a separate job, executed in parallel with `test`, is probably minuscule (or maybe even the performance impact is negative)

Reply:
Yeah, compile depends on `scalafmtCheckAll` and `scalafixAll --check` depends on compile